e2e/config.go

```diff
@@ -37,8 +39,28 @@ var (
 	lvmOperatorUninstall bool
 	scheme               = runtime.NewScheme()
 	crClient             crclient.Client
+	ClientSet            clientset.Interface
+	parameterCodec       runtime.ParameterCodec
 )
 
+type Interface interface {
```

Comment:
Changes in this file doesn't look right.  Looks right the code from the library!

Reply:
Updated.